src/LanguageServer/Impl/Implementation/Server.Editor.cs

```diff
@@ -14,11 +14,13 @@
 // permissions and limitations under the License.
 
 using System;
+using System.ComponentModel.Design;
```

Comment:
What is this using for?

Reply:
Auto-inserted by resharper I guess. Not used, cleaned up.